Cargo.toml

```diff
@@ -104,6 +104,22 @@ members = [
     "src/regexp",
 ]
 
+# Here we should specify our high level binaries to prevent folks from needing to take extra steps
+# like updating git submodules when building the workspace without a target specified.
+#
+# Crates we want to make sure are _excluded_ are:
+# * `fivetran-destination` since it depends on the `fivetran-sdk` submodule.
+#
+default-members = [
```

Comment:
`mz` probably needs to be here too?

Reply:
Updated to include everything that's referenced from an `mzbuild.yml`